key_reader.py

```diff
@@ -7,16 +7,22 @@
 def read_keys():
     keys_dict = {}
     keys_file = open(KEYS_PATH, "r")
-    for line in keys_file:
-        key, value = line.strip().split(',')
-        keys_dict[key] = value
+    try:
+        for line in keys_file:
+            key, value = line.strip().split(',')
+            keys_dict[key] = value
+    except ValueError:
+        raise ValueError(f"File {KEYS_PATH} has an incorrect format.\n"
+                         "It should be a comma-delimited text, "
+                         "following the scheme:\n"
+                         "\"<key-name>,<key-value>\"")
 
     keys_file.close()
 
     if 'weatherapi.com' not in keys_dict or 'meteostat.net' not in keys_dict:
-        raise KeyError("""The API key names in the better_weather_keys.txt
-                       file are missing or their names might be incorrect.
-                       The correct names are: 'weatherapi.com', 'meteostat.net'""")
+        raise KeyError("The API key names in the better_weather_keys.txt "
+                       "file are missing or their names might be incorrect. "
+                       "The correct names are: 'weatherapi.com', 'meteostat.net'")
```

Comment:
```
KeyError: "The API key names in the better_weather_keys.txt file are missing or their names might be incorrect. The correct names are: 'weatherapi.com', 'meteostat.net'"
```
I suggest adding `\n` after the `incorrect` word to have a similar pattern as for the keys file error (if the previous suggestion is applied) and forecast period error:
```
KeyError: "The API key names in the better_weather_keys.txt file are missing or their names might be incorrect.
The correct names are: 'weatherapi.com', 'meteostat.net'"
```

Reply:
@KrzysiekJodlowski 
This is the one example that I was mentioning during our meeting, for which '\n' is not working.
`KeyError: "The API key names in the better_weather_keys.txt file are missing or their names might be incorrect.\n The correct names are: 'weatherapi.com', 'meteostat.net'"`
So I am leaving it as it is now.